Deploy/bash/geteob.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Set to the name of the service you're testing
+SANDBOX=http://sandbox.ab2d.cms.gov/api/v1/fhir
+
+# Set up the auth to use for Okta
+AUTH=MG9hMnQwbHNyZFp3NXVXUngyOTc6SEhkdVdHNkxvZ0l2RElRdVdncDNabG85T1lNVmFsVHRINU9CY3VIdw==
```

Comment:
This requires the user of this script to figure out the base64-encoded authorization header, which is slightly inconvenient albeit not critical. Should the script rather take in client id & client secret instead? The `Authorization` header can be derived in-script via ```export AUTH=`echo -n client_id-value:client_pass_value | base64` ```

Reply:
I actually put the Base64 encoding for all the values in the API Use confluence page. I had originally used the client id and password that, but I was getting a different second to last character than when it expected using the bas64 command in my script. I'm not entirely sure why that was happening, so to eliminate any similar issues for the person using the sandbox, I just put all the encoded values on the web page.